package.json

```diff
@@ -82,16 +82,18 @@
   ],
   "jspm": {
     "main": "js/bootstrap",
+    "ignore": ["dist/js/npm"],
+    "directories": {
+      "lib": "dist"
```

Comment:
Yes it's best to use the existing dist folder - for example lots of users use jspm without any transpiler enabled and just to manage ES5 module formats. It's still the lowest common denominator for a little while yet.